alacritty/src/event.rs

```diff
@@ -1546,6 +1505,30 @@ impl Processor {
 
                     info!("Initialisation complete");
                 },
+                // NOTE: Request a redraw for terminal wakeup to minimize the input latency.
+                WinitEvent::UserEvent(Event {
+                    window_id: Some(window_id),
+                    payload: EventType::Terminal(TerminalEvent::Wakeup),
+                }) => {
+                    if let Some(window_context) = self.windows.get_mut(&window_id) {
+                        window_context.dirty = true;
+                        if window_context.display.window.has_frame {
+                            window_context.display.window.request_redraw();
+                        }
+                    }
+                },
+                // NOTE: Handle here to not delay rendering much.
```

Comment:
Same as the previous `NOTE` label comment above.

Also handle here what? This comment is not providing any information at all.